src/test/java/nl/tudelft/contextproject/controller/ControllerTest.java

```diff
@@ -127,6 +133,7 @@ public void testRemoveDrawable() {
 		Node rn = mock(Node.class);
 		c.setRootNode(rn);
 		Geometry geom = mock(Geometry.class);
+
 		Drawable d = new Drawable() {
```

Comment:
While not necessarily related to this PR, we usually do something like this with a mock.